### ISSUE TITLE
About the error "assert os.path.isfile(_image) AssertionError"

### ISSUE THREAD

**Issue by Juliano-yotto** [VainF/DeepLabV3Plus-Pytorch#117]
Hi, I try to run a new dataset but got this error : 
File "E:\pycharm_code\deeplabV3+\pytorch-deeplab-xception-master\dataloaders\datasets\invoice.py", line 52, in __init__
assert os.path.isfile(_image)
AssertionError
format of train images——.jpg，labels——.png
hope you can help me out
thanks

**Comment by Juliano-yotto** [comment]
It's already resolved

**Comment by shihenren** [comment]
> It's already resolved

how you solve it?

**Comment by ZX815** [comment]
How to solve this problem?

**Comment by JingyuCheng1014** [comment]
how you solve it?
